packages/ui/src/contexts/attachment.tsx

```diff
@@ -82,7 +83,7 @@ export const AttachmentProvider = ({
   children,
 }: PropsWithChildren<{
   canUpload: boolean;
-  uploadAsset: (asset: ImagePickerAsset) => Promise<void>;
+  uploadAsset: (asset: ImagePickerAsset, isWeb?: boolean) => Promise<void>;
```

Comment:
Possible to skip this arg? We could prettily easily add an `isWeb` to the shared package just by having an `isWeb.ts` and `isWeb.native.ts`.